lib/modules/ssh/strong_password/__init__.py

```diff
@@ -13,5 +30,7 @@ def module_configuration():
     """
     return {
         "username": "root",
-        "password": generate_token(16)
+        "password": generate_token(16),
+        "extra_docker_options": [],
```

Comment:
share the file for farther process `"extra_docker_options": ["-v tmp/ssh_strong_password_logins.txt:/root/logs/logins.txt"],`

Reply:
yup would add this